doc/tutorials/Content_Page.md

```diff
@@ -0,0 +1,86 @@
+# Content Page
```

Comment:
Is the `depend` part of the `dart_package_install_depend.PNG` name redundant? Sorry if I'm wrong.

Reply:
Let me rename it to `dart_package_install.png`